packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -394,7 +460,7 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
     }
 
     .mdc-textfield__label {
-      color: $mdc-textfield-error-on-light;
+      @include mdc-theme-prop(color, $mdc-textfield-error-on-light);
```

Comment:
We need to port the `::after` invalid styles above and below this to apply to the bottom-line element instead. The `::after` styles no longer have any effect on regular textfields, and so the bottom line is no longer red whereas it was before.

Oddly, the bottom line isn't red on textfield-box anymore either...

Reply:
Fixed it